pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -198,6 +199,8 @@ const ServiceAnnotationLoadBalancerHCTimeout = "service.beta.kubernetes.io/aws-l
 // service to specify, in seconds, the interval between health checks.
 const ServiceAnnotationLoadBalancerHCInterval = "service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval"
 
+const labelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
I think this const is already available somewhere in the codebase (I've seen it before)

Reply:
It's in a few places, but nothing we'd want to import in here AFAICT:

* `cmd/kubeadm/app/constants/constants.go`
* `pkg/controller/service/service_controller.go`

Reply:
Makes sense, thanks